### PR TITLE
Style output table

### PR DESCRIPTION
**Pull request by fraoustin**
## Description

add color for table. You can use a Style with token 

> class YourStyle(Style):
>        default_style = ""
>        styles = {
>            Token.Output.SeparatorTable : '#ansipurple'
>        }

It's a same principe that style_output for the contents of the table

## Checklist

- [ ] I've added this contribution to the `CHANGELOG`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by fraoustin** [comment]
there is a problem with travis: "oci runtime error: exec failed: container_linux.go:265: starting container process caused could not create session key: disk quota exceeded"
Can you see this?

**Comment by meeuw** [comment]
@fraoustin I think there was an issue with travis, I've restarted the failed jobs and they are all :ok: now

**Comment by fraoustin** [comment]
thank you

**Comment by fraoustin** [comment]
it's ok for you?

File: cli_helpers/tabular_output/output_formatter.py [hunk 1]
```diff
@@ -209,11 +209,13 @@ def format_output(data, headers, format_name, **kwargs):
 for tabulate_format in tabulate_adapter.supported_formats:
     TabularOutputFormatter.register_new_formatter(
         tabulate_format, tabulate_adapter.adapter,
-        tabulate_adapter.preprocessors,
+        tabulate_adapter.preprocessors +
+        (tabulate_adapter.style_output_table(tabulate_format),),
```
Comment (tsroten) [inline cli_helpers/tabular_output/output_formatter.py]: If you wanted to, you could pass `format_name` in via [`format_output`](https://github.com/dbcli/cli_helpers/pull/38/files#diff-54f002145bfa7a641b8eccca6ce5032aR146) so you wouldn't have to handle the different formats here. It's fine as is, too, though.

**Comment by tsroten** [comment]
@fraoustin Once you update the changelog, we can get this merged 😄

**Comment by jayvdb** [comment]
ping @fraoustin

**Comment by tsroten** [comment]
Thanks @fraoustin!!

**Pull request merged**